DentOS_Framework/DentOsTestbed/src/dent_os_testbed/Device.py

```diff
@@ -76,13 +76,24 @@ def __init__(self, logger, loop, params):
             self.files_to_collect = []
             # dictionary of links with device name as key
             self.links_dict = {}
+            # meida mode can be: fiber, mixed, or copper
+            # If mixed then all even vports are
+            # assumed to be copper and odd as fiber
```

Comment:
Please correct me If I'm wrong but you read explicitly provided media mode in TG ports instead of separation to odd and even.

Reply:
done